### PR TITLE
Cogex web app

### PR DESCRIPTION
**Pull request by kkaris**
This PR sets up a web app with Flask that ties together several apps:

- The gene/metabolite list analysis
- The data display app that serves the standalone statement view page and the go curation interface
- The query backend with api docs
- The landing page linking to the other apps

To test locally:

```sh
python -m indra_cogex.apps -vv
```

Todo for final review:
---------------------------

- [x] Landing page needs descriptions ~and images~ for the linkouts
- [x] Separate files for helper functions
- [x] Separate files for app code/blueprints
- [x] Separate files for cli/run command
- [x] Write docstrings for all public functions/methods

## Homepage Screenshot
Screen Shot 2022-03-22 at 14 59 18

### CODE REVIEW

File: src/indra_cogex/apps/landing_page/__init__.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        "home.html",
+        format_number=_figure_number,
+        node_counter=node_counter,
+        edge_counter=edge_counter,
+        edge_labels=edge_labels,
+        blueprints=current_app.blueprints,
+    )
+
+
+app = flask.Flask(__name__, template_folder=TEMPLATES_DIR, static_folder=STATIC_DIR)
```
Comment (cthoyt) [inline src/indra_cogex/apps/landing_page/__init__.py:83]: this app should be in indra_cogex.apps.landing_page.cli so it doesn't get created if you import this blueprint for a different app

Comment (kkaris) [reply src/indra_cogex/apps/landing_page/__init__.py:83]: Good point

**Review by cthoyt — APPROVED** [review]

File: src/indra_cogex/apps/data_display/cli.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from flask import Flask
+from flask_bootstrap import Bootstrap4
+from indralab_auth_tools.auth import auth, config_auth
+from more_click import make_web_command
+
+from indra_cogex.apps.constants import INDRA_COGEX_EXTENSION, STATIC_DIR, TEMPLATES_DIR
+from indra_cogex.apps.data_display import data_display_blueprint
+from indra_cogex.client.neo4j_client import Neo4jClient
+
+app = Flask(__name__, template_folder=TEMPLATES_DIR, static_folder=STATIC_DIR)
```
Comment (bgyori) [inline src/indra_cogex/apps/data_display/cli.py:10]: We could add
```
app.config['SWAGGER_UI_DOC_EXPANSION'] = 'list'
```

**Review by bgyori — APPROVED** [review]
Awesome!

**Pull request merged**